examples/managed_node_groups/main.tf

```diff
@@ -113,7 +113,6 @@ module "eks" {
         ExtraTag = "example"
       }
     }
-    defaults = {}
```

Comment:
@dpiddockcmp is it OK remove this? Without the example creates 2 node groups where I think it should be a single?

Reply:
Should be fine. It was for testing some of the for_each and ternary logic. Terraform gets more unpredictable with multiple items. Used to confirm the fix in #633